### PR TITLE
add eventValue, SDK links and custom config

### PR DESCRIPTION
**Pull request by mauerbac**

### CODE REVIEW

**Comment by mikeng13** [comment]
LGTM

File: src/content/server/reference/custom/2-logger.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        datafile: datafile,
+        eventDispatcher: defaultEventDispatcher,
+        logger: defaultLogger.createLogger(),
+      });
+---
+
+You can provide a logging method that logs messages when certain events occur in the SDK. You can write your own logger if you would like to customize things like message formatting or minimum logging levels. 
+
+
+By default, each SDK assumes no-op logging if a custom logger isn’t provided. To improve your experience in setting up the SDK, it’s recommended that you pass in a logger when creating an `Optimizely` instance.
```
Comment (aliabbasrizvi) [inline src/content/server/reference/custom/2-logger.md]: We also provide SimpleLogger out of the box. Like in Python here: https://github.com/optimizely/python-sdk/blob/master/optimizely/logger.py#L19

Comment (aliabbasrizvi) [inline src/content/server/reference/custom/2-logger.md]: And here in Ruby: https://github.com/optimizely/ruby-sdk/blob/master/lib/optimizely/logger.rb#L18

File: src/content/server/reference/custom/2-logger.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+<div class="hidden visible" data-toggle-section="java-code">
+For the Java SDK, we require the use of an [SLF4J](http://www.slf4j.org) implementation. No simple logger is provided.
+</div>
+
+There are 4 log levels used in the SDKs: `debug`, `info`, `warning`, and `error`.
```
Comment (aliabbasrizvi) [inline src/content/server/reference/custom/2-logger.md]: There are different log levels. Depends on the SDK.

File: src/content/server/reference/custom/2-logger.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+A few examples of log messages --
+<br />**Debug logs:** impression event dispatched & conversion event dispatched
```
Comment (aliabbasrizvi) [inline src/content/server/reference/custom/2-logger.md]: This happens at Info level also. We print information like bucketing value at this level.

File: src/content/server/reference/custom/3-error.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+      optimizely = optimizely.createInstance({
+        datafile: datafile,
+        errorHandler: defaultErrorHandler,
+        eventDispatcher: defaultEventDispatcher,
+        logger: defaultLogger.createLogger(),
+      });
+---
+
+You can provide your own custom error handler logic by overriding the `handle_error/handleError` function, which will allow you to standardize error handling across your production environment.
```
Comment (aliabbasrizvi) [inline src/content/server/reference/custom/3-error.md]: Like logger we also have a raise all exception error handler as well.

File: src/content/server/reference/custom/2-logger.md (hunk 2 above)
Comment (mauerbac) [inline src/content/server/reference/custom/2-logger.md]: For now, I'll say "depending on the SDK there are different log levels" but we should be more explicit and list per sdk in the future

**Pull request merged**